packages/jest-worker/src/workers/__tests__/processChild.test.ts

```diff
@@ -226,110 +203,87 @@ it('returns results immediately when function is synchronous', () => {
+    '"null" or "undefined" thrown',
+  );
 
-  expect(process.send).toHaveBeenCalledTimes(5);
+  expect(spyProcessSend).toHaveBeenCalledTimes(5);
 });
 
 it('returns results when it gets resolved if function is asynchronous', async () => {
   process.emit('message', [
     CHILD_MESSAGE_INITIALIZE,
-    true, // Not really used here, but for flow type purity.
+    true,
     './my-fancy-worker',
   ]);
 
-  process.emit('message', [
-    CHILD_MESSAGE_CALL,
-    true, // Not really used here, but for flow type purity.
-    'fooPromiseWorks',
-    [],
-  ]);
+  process.emit('message', [CHILD_MESSAGE_CALL, true, 'fooPromiseWorks', []]);
 
   await sleep(10);
 
-  expect(process.send.mock.calls[0][0]).toEqual([PARENT_MESSAGE_OK, 1989]);
+  expect(spyProcessSend.mock.calls[0][0]).toEqual([PARENT_MESSAGE_OK, 1989]);
 
-  process.emit('message', [
-    CHILD_MESSAGE_CALL,
-    true, // Not really used here, but for flow type purity.
-    'fooPromiseThrows',
-    [],
-  ]);
+  process.emit('message', [CHILD_MESSAGE_CALL, true, 'fooPromiseThrows', []]);
 
   await sleep(10);
 
-  expect(process.send.mock.calls[1][0]).toEqual([
+  expect(spyProcessSend.mock.calls[1][0]).toEqual([
     PARENT_MESSAGE_CLIENT_ERROR,
     'TypeError',
-    'Booo',
+    'Boo',
     mockError.stack,
     {},
   ]);
 
-  expect(process.send).toHaveBeenCalledTimes(2);
+  expect(spyProcessSend).toHaveBeenCalledTimes(2);
 });
 
 it('calls the main module if the method call is "default"', () => {
   process.emit('message', [
     CHILD_MESSAGE_INITIALIZE,
-    true, // Not really used here, but for flow type purity.
+    true,
     './my-fancy-standalone-worker',
   ]);
 
-  process.emit('message', [
-    CHILD_MESSAGE_CALL,
-    true, // Not really used here, but for flow type purity.
-    'default',
-    [],
-  ]);
+  process.emit('message', [CHILD_MESSAGE_CALL, true, 'default', []]);
 
-  expect(process.send.mock.calls[0][0]).toEqual([PARENT_MESSAGE_OK, 12345]);
+  expect(spyProcessSend.mock.calls[0][0]).toEqual([PARENT_MESSAGE_OK, 12345]);
 });
 
 it('calls the main export if the method call is "default" and it is a Babel transpiled one', () => {
   process.emit('message', [
     CHILD_MESSAGE_INITIALIZE,
-    true, // Not really used here, but for flow type purity.
+    true,
     './my-fancy-babel-worker',
   ]);
 
-  process.emit('message', [
-    CHILD_MESSAGE_CALL,
-    true, // Not really used here, but for flow type purity.
-    'default',
-    [],
-  ]);
+  process.emit('message', [CHILD_MESSAGE_CALL, true, 'default', []]);
 
-  expect(process.send.mock.calls[0][0]).toEqual([PARENT_MESSAGE_OK, 67890]);
+  expect(spyProcessSend.mock.calls[0][0]).toEqual([PARENT_MESSAGE_OK, 67890]);
 });
 
 it('removes the message listener on END message', () => {
   // So that there are no more open handles preventing Node from exiting
   process.emit('message', [
     CHILD_MESSAGE_INITIALIZE,
-    true, // Not really used here, but for flow type purity.
```

Comment:
can keep this comments, but just drop `flow` - `// Not really used here, but for type purity.`